client/src/contexts/UserProvider.js

```diff
@@ -5,14 +5,31 @@ const UserProvider = ({ children }) => {
     const [user, setUser] = useState({});
 
     useEffect(() => {
-        fetch("/api/auth/users/me")
-            .then(res => {
-                res.json()
-            })
-            .then(res => setUser(res))
-            .catch(err => {
-                console.log(err);
+        fetch("http://localhost:3001/api/auth/users/me", {
+            method: "GET",
+            credentials: "include",
+            headers: {
+                Accept: "application/json",
+                "Content-Type": "application/json",
+                "Access-Control-Allow-Credentials": true
+            }
+        })
+        .then(response => {
```

Comment:
@bonnieli I've added these properties to the fetch request in the UserProvider context.

Reply:
This is what the user state has been set to after calling the fetch promise
```
{
  "authenticated": true,
  "user": {
    "availabilities": [],
    "meetingTypes": [],
    "googleId": "104348754318193001966",
    "firstName": "Michael",
    "lastName": "S",
    "email": "singh.michaelanthony@gmail.com",
    "profilePicture": "https://lh3.googleusercontent.com/a-/AOh14GjAmMF8c19hWYlzl5fdYAyn5VS0iKNv95ugXtl_Pw=s96-c",
    "createdAt": "2020-11-20T18:22:07.872Z",
    "updatedAt": "2020-11-20T18:22:07.872Z",
    "id": "5fb8094f8c6825146949b002"
  }
}
```